CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ## Changed
 
+# 0.0.!! (2023-05-08 / ???)
```

Comment:
Please remove this line, just add an entry to one of the three categories under "unreleased", the tooling will do the rest.

Reply:
Removed in latest.